rules/legacy/tests/Rector/Include_/AddTopIncludeRector/Expected/has_include.php.inc

```diff
@@ -1,13 +0,0 @@
-<?php
-
-namespace Rector\Legacy\Tests\Rector\Include_\AddTopIncludeRector\Fixture;
-
-class SomeClass
-{
-
-    private function someMethod() : void
-    {
-        include 'somefile.php';
-    }
-
-}
```

Comment:
If the file doesn't change at all, we only test the file is the same as input file. So this fixture can be removed

Reply:
Actually, this fixture was testing false postive for `Class_`. The method that looked for include was never called.

I've chaged it only to function, so the `require_once` is checked